contracts/keeper/CollateralizationOracleKeeper.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.8.0;
+
+import "../token/FeiTimedMinter.sol";
+import "../oracle/ICollateralizationOracleWrapper.sol";
+
+/// @title CollateralizationOracleKeeper
+/// @notice a FEI timed minter which only rewards when updating the collateralization oracle 
+contract CollateralizationOracleKeeper is FeiTimedMinter {
```

Comment:
Did not add a unit test for this class because it is such as simple override of the FeiTimedMinter.